src/Admin/AdminPage.php

```diff
@@ -64,11 +66,71 @@ public function add_admin_menu() {
     }
 
     /**
-     * Render the main page content.
+     * Enqueue CSS and JS for the admin pages.
+     */
+    function enqueue_admin_assets() {
+        // Enqueue CSS
+        wp_enqueue_style(
+            'malware-scanner-admin-css',  // Handle
+            plugin_dir_url(__FILE__) . 'assets/css/admin-styles.css',  // Path to CSS file
+            [],  // Dependencies
+            '1.0'  // Version
+        );
+    
+        // Enqueue JS
+        wp_enqueue_script(
+            'malware-scanner-admin-js',  // Handle
+            plugin_dir_url(__FILE__) . 'assets/js/admin-scripts.js',  // Path to JS file
+            ['jquery', 'chart-js'],  // Dependencies (jQuery, Chart.js)
+            '1.0',  // Version
+            true  // Load in footer
+        );
+    }
+    
+    function enqueue_font_awesome_admin() {
+        // Enqueue Font Awesome for admin area
+        wp_enqueue_style('font-awesome-admin', 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css', array(), null);
```

Comment:
It would be great if we are using https://developer.wordpress.org/resource/dashicons/ instead of font-awesome to avoid resource loading

Reply:
@rajanvijayan Please review this PR